chrome/content/zotero/xpcom/attachments.js

```diff
@@ -722,23 +722,31 @@ Zotero.Attachments = new function(){
 	 */
 	this.importFromDocument = Zotero.Promise.coroutine(function* (options) {
 		Zotero.debug('Importing attachment from document');
-		
+
 		var libraryID = options.libraryID;
 		var document = options.document;
 		var parentItemID = options.parentItemID;
 		var title = options.title;
 		var collections = options.collections;
 		var saveOptions = options.saveOptions;
+		var content = options.content;
+		var url = options.url;
+		var contentType = options.contentType;
```

Comment:
I assume these lines and the conditional below were from an earlier version (before `importFromPageData()`?) and can be removed?